<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout" stretch="1,0">
     <item>
      <layout class="QGridLayout" name="diceGridLayout">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="d4HLayout">
           <item>
            <widget class="QSpinBox" name="d4SpinBox"/>
           </item>
           <item>
            <widget class="QRadioButton" name="d4RadioButton">
             <property name="text">
              <string>d4</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="d6HLayout">
           <item>
            <widget class="QSpinBox" name="d6SpinBox"/>
           </item>
           <item>
            <widget class="QRadioButton" name="d6RadioButton">
             <property name="text">
              <string>d6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="d8HLayout">
           <item>
            <widget class="QSpinBox" name="d8SpinBox"/>
           </item>
           <item>
            <widget class="QRadioButton" name="d8RadioButton">
             <property name="text">
              <string>d8</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="d10HLayout">
           <item>
            <widget class="QSpinBox" name="d10SpinBox"/>
           </item>
           <item>
            <widget class="QRadioButton" name="d10RadioButton">
             <property name="text">
              <string>d10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="d12HLayout">
           <item>
            <widget class="QSpinBox" name="d12SpinBox"/>
           </item>
           <item>
            <widget class="QRadioButton" name="d12RadioButton">
             <property name="text">
              <string>d12</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <layout class="QHBoxLayout" name="d20HLayout">
           <item>
            <widget class="QSpinBox" name="d20SpinBox"/>
           </item>
           <item>
            <widget class="QRadioButton" name="d20RadioButton">
             <property name="text">
              <string>d20</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="d100HLayout">
           <item>
            <widget class="QSpinBox" name="d100SpinBox"/>
           </item>
           <item>
            <widget class="QRadioButton" name="d100RadioButton">
             <property name="text">
              <string>d100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Add (to each:)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="eachAddSpinBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Roll:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Add (to total):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="diceSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="totalAddSpinBox"/>
       </item>
       <item row="1" column="3">
        <layout class="QVBoxLayout" name="buttonLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="runButton">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="statsForm">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="medianLabel">
         <property name="text">
          <string>Median</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="meanLabel">
         <property name="text">
          <string>Mean</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="modeLabel">
         <property name="text">
          <string>Mode</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="medianValue">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="meanValue">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="modeValue">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
